lib/domain.js

```diff
@@ -387,3 +382,51 @@ Domain.prototype.bind = function(cb) {
 
   return runBound;
 };
+
+// communicate with events module, but don't require that
```

Comment:
We should change this comment. It serves no purpose now, other than backward compatibility.

Reply:
Updated!